ui/app/AppMain.qml

```diff
@@ -350,76 +350,49 @@ Item {
         appView: StackLayout {
             id: appView
             anchors.fill: parent
-            currentIndex: {
```

Comment:
@alexandraB99 why did you remove this? Please analyze again this code. Actually changes needed due to changed model, which is used for displaying chats/categories/channels, do not have anything and don't affect anyhow this part here.